index-de.html

```diff
@@ -29,89 +29,57 @@ <h1 id="top">Ein grafischer Spickzettel für Git</h1>
     </ul>
   </div>
 
-  <p id="link-to-png">
-    Sollten die
-    <a href="http://de.wikipedia.org/wiki/Scalable_Vector_Graphics">SVG</a>
-    Vektorgrafiken nicht angezeigt werden, kann die
-    <a
-    href="?no-svg">SVG freie</a>
-    Variante gewählt werden.
-  </p>
+  <p id="link-to-png">Sollten die Grafiken nicht funktionieren, kannst du die
+    <a href="?no-svg">SVG-freie</a> Version dieser Seite auswählen.</p>
 
-  <p id="link-to-svg">
-    SVG Vektorgrafiken sind deaktiviert.
-    <a href="index.html">(SVG aktivieren)</a>
-  </p>
+  <p id="link-to-svg">SVG-Grafiken sind abgeschaltet.
+    <a href="index.html">(SVG wieder aktivieren)</a></p>
```

Comment:
@HybridDog Very helpful feedback, thank you! I'll incorporate it in my next commits.